manifest.php

```diff
@@ -27,7 +27,7 @@
     'label' => 'LTI Delivery Tool Provider',
     'description' => 'The LTI Delivery Tool Provider allows third party applications to embed deliveries created in Tao',
     'license' => 'GPL-2.0',
-    'version' => '10.0.1',
+    'version' => '10.1.0',
```

Comment:
Could you please increase minimum required tao version to 39.4.0 (with required functionality in ActionResolver)